spec/requests/queues/all_queued_spec.rb

```diff
@@ -31,14 +31,29 @@
                       updated_at: 3.days.ago)
   end
 
-  it 'shows all the queued items' do
-    get '/workflow_queue/all_queued?repository=dor&limit=3&hours-ago=24'
-
-    expect(response.body).to be_equivalent_to <<~XML
-      <workflows>
-        <workflow name="accessionWF" process="shelve" druid="#{one.druid}" laneId="default"/>
-        <workflow name="accessionWF" process="shelve-complete" druid="#{two.druid}" laneId="fast"/>
-      </workflows>
-    XML
+  context 'with repository query arg' do
+    it 'shows all the queued items' do
+      get '/workflow_queue/all_queued?repository=dor&limit=3&hours-ago=24'
+
+      expect(response.body).to be_equivalent_to <<~XML
+        <workflows>
+          <workflow name="accessionWF" process="shelve" druid="#{one.druid}" laneId="default"/>
+          <workflow name="accessionWF" process="shelve-complete" druid="#{two.druid}" laneId="fast"/>
+        </workflows>
+      XML
```

Comment:
does it make sense for this to be in a let(:expected_xml) clause, partly to emphasis that the response is the same in both cases?

Reply:
dig it, will do